browser/ipfs/content_browser_client_helper.cc

```diff
@@ -52,6 +52,9 @@ bool HandleIPFSURLRewrite(GURL* url, content::BrowserContext* browser_context) {
        base::EndsWith(url->host_piece(), kIpnsLocalhost))) {
     return true;
   }
+  if (!brave::IsRegularProfile(browser_context)) {
```

Comment:
should this go before https://github.com/brave/brave-core/pull/14147/files#diff-c303072583404f1b371ada943d4968bd01fc742d5403b7a6612e7765accb11b8R50 ?